finance/jobs/2020_githubresponder.rst

```diff
@@ -0,0 +1,25 @@
+GitHub First/Last Responder
+===========================
+
+The Astropy Project is looking to hire a Github first responder / last responder. The overarching goal of this position is to improve the first time contributor experience, reduce the technical barriers to contributions, reduce the number of open issues, and increase the number of merged pull requests. This is not intended to cover all issues and PRs opened in astropy, but rather to support and lighten the load on the rest of the volunteer community.
+An ideal candidate would be familiar with the astropy project ecosystem and capabilities, comfortable with the Github Development workflow, and have experience providing supportive feedback and technical advice in online environments.
+
+This role includes  monitoring  the astropy issue tracker, add labels to new issues, provide a friendly welcome message on issues and pull requests, and give starting points for how to resolve the issue or move the pull request forward. The “last responder” aspect of the role includes helping to close out issues and PRs, helping with rebasing, and troubleshooting failing tests.
+
+We expect a commitment of about 8-10 hours/week for 12 months. The Astropy Project can pay inside or outside of the US. Compensation could be provided as a consulting fee or as a subaward. This is a remote-only position.
+
+Please send a short (one paragraph) email to finance@astropy.org with a description of your interest in the role, your qualifications, and any relevant experience.  If possible, please provide links to a few examples of issues/PRs where you have been involved as a fist and/or last responder.
```

Comment:
I'd also include a request for a CV

Reply:
References would be nice too, if that is not too much to ask.

Reply:
The intention is that applying (and reviewing) is easy. This is intended (by CoCo decision) to be open to current astropy-maintainers only. We know those people and their experience. If noone steps forward, and we need to open this to the wider community, then we'll need to ask for more extensive documentation (CV, references).

Reply:
oh, that's answers my biggest concern then (e.g. how would this ever work with a brand new person, as I don't feel this is the limiting bottleneck and not unavailable maintainer reviews), thanks @hamogu for the clarification.